src/rules/preferForOfRule.ts

```diff
@@ -17,7 +17,7 @@
 
 import * as ts from "typescript";
 import * as Lint from "../index";
-import { isAssignment } from "../language/utils";
+import { isAssignment, unwrapParentheses } from "../language/utils";
```

Comment:
Good point, but that was mainly done for expediency when transitioning the codebase from namespaces to modules -- it was easiest to reference things on the `Lint` object. Now that we have modules, it would be nice to slowly move to more granular imports.